src/components/Schedule/Schedule.tsx

```diff
@@ -94,6 +94,682 @@ class Schedule extends React.Component<ISentinelProps, ISentinelState> {
             conditionSectionActive ? '' : ' main-section-blue'
           }`}
         >
+          <div className={`bx--col-xs-6 main-section`}>
```

Comment:
I'll probably refactor this later but these changes should be extracted into a separate component because this file is getting huge, and also to achieve a nice separation of concerns. This is more of an FYI but you can try to do it if you want.

Reply:
Yes, that's what I thought, it should a separate .tsx file. But I wouldn't dare to do it myself without a full understanding of the functions above **public render()**